frontend/src/router/index.ts

```diff
@@ -0,0 +1,27 @@
+import Vue from 'vue';
+import VueRouter, { RouteConfig } from 'vue-router';
+import NewGame from '../views/NewGame.vue';
+import InGame from '../views/InGame.vue';
+
+Vue.use(VueRouter);
+
+const routes: RouteConfig[] = [
+  {
+    path: '/',
+    name: 'NewGame',
+    component: NewGame,
+  },
+  {
+    path: '/:gameId',
```

Comment:
how about being restful?

```suggestion
    path: '/games/:gameId',
```